source/consumers/elastic/media-insights-elasticsearch.yaml

```diff
@@ -64,14 +65,13 @@ Resources:
         Version: "2012-10-17"
         Statement:
           - Effect: Allow
-            Principal: "*"
+            Principal:
+              AWS:
+                - !Ref MIEAdminRoleArn
             Action:
-              - es:*
-            Condition:
-              IpAddress:
-                aws:SourceIp: !Ref AccessCidr
+              - "es:ESHttpGet"
```

Comment:
I also updated our access to only allow HttpGet, to help prevent any bad data from being inserted into the cluster. There really shouldn't be any PUT/POST access into the cluster from the webapp anyways.